script/bootstrap

```diff
@@ -4,9 +4,10 @@ set -e
 export CC=gcc
 
 echo "==> Installing gem dependencies…"
+gem install bundle
```

Comment:
Can this be changed to:

```
which bundle || gem install bundle
```

That will prevent reinstallation of bundler if it's already installed.